### ISSUE TITLE
BatchSize set to 1

### ISSUE THREAD

**Issue by XiShuFan** [limhoyeon/ToothGroupNetwork#23]
Hello! I have a question for your inspiring work. Did you set batch size to 1 when training? Cause I find that you use BatchNorm in your network

**Comment by limhoyeon** [comment]
Yes  I set batch size 1 which would be the same as layer normalization... 
Due to my limited gpu vram, i wanna set more batch size but i couldn`t do that..
